python/GafferUI/CompoundEditor.py

```diff
@@ -124,6 +124,49 @@ def restoreWindowState( self ) :
 		for p in self.__detachedPanels :
 			p.restoreWindowState()
 
+	# Returns the editor of the specified type that has the mouse focus. This
+	# takes into account detached panels and window ordering such that when the
+	# mouse is not over an editor of the requested type, in the active window,
+	# the first editor in the active window belonging to the CompoundEditor will be
+	# returned. If none can be found, then the first matching editor from any
+	# window will be returned.
+	def focusedEditor( self, type = GafferUI.Editor, fallbackToFirstOfType = False ) :
```

Comment:
- Move this function underneath the `editors()` function, since they are similar in purpose.
- focused -> focus**s**ed
- Can we consider something shorter than `fallbackToFirstOfType`? Just `fallbackToFirst` or `fallbackToUnfocussed` perhaps? Or even just `fallback`?

Reply:
All fixed in https://github.com/GafferHQ/gaffer/pull/3372/commits/5129e4597c2dc9c844e6e05e387032c3e65588fc